docs/admin/metadata.mdx

```diff
@@ -53,7 +53,6 @@ The following options are available for Root Metadata:
 | Key | Type | Description |
 | --- | --- | --- |
 | `defaultOGImageType` | `dynamic` (default), `static`, or `off` | The type of default OG image to use. If set to `dynamic`, Payload will use Next.js image generation to create an image with the title of the page. If set to `static`, Payload will use the `defaultOGImage` URL. If set to `off`, Payload will not generate an OG image. |
-| `icons` | `IconConfig[]` | An array of icon objects. [More details](#icons). |
```

Comment:
Is there anything that's changed about this in this PR? Or is this left over from the previous PR?